lib/registration.pm

```diff
@@ -764,7 +764,7 @@ sub get_addon_fullname {
         hpcm => 'sle-module-hpc',
         legacy => 'sle-module-legacy',
         lgm => 'sle-module-legacy',
-        ltss => 'SLES-LTSS',
+        ltss => is_hpc() ? 'SLE_HPC-LTSS' : 'SLES-LTSS',
```

Comment:
The logic is not make sense, not hpc then is sles? We couldn't get this conclusion.